### PR TITLE
Add clearer SDK versions for paywalls

### PR DESCRIPTION
**Pull request by joshdholtz**
## Motivation / Description

It wasn't very clear which SDKs (and their versions) all supported paywalls

## Changes introduced

Created a table at the top of the main page (similar to Offering Metadata) to list the SDKs and their versions that paywalls is supported under

### CODE REVIEW

File: docs_source/🛠 Tools/paywalls.md [hunk 1]
```diff
@@ -5,9 +5,17 @@ hidden: false
 ---
 RevenueCat's Paywalls allow you to remotely configure your entire paywall view without any code changes or app updates. Whether you’re building a new app, exploring new paywall concepts, or diving into experimentation; RevenueCat’s Paywalls make it easy to get started.
 
+## Paywalls are supported in the following SDK versions:
+| RevenueCat SDK           | Version required for Offering Metadata |
```
Comment (NachoSoto) [inline docs_source/🛠 Tools/paywalls.md:9]: Should this be "for Paywalls"?

File: docs_source/🛠 Tools/paywalls.md [hunk 2]
```diff
@@ -82,6 +90,14 @@ implementation 'com.revenuecat.purchases:purchases-ui:7.1.0'
 > It is safe to release app updates with it. We guarantee that paywalls will continue to work and any changes will always be backwards compatible.
 > They are stable, but migration steps may be required in the future. We'll do our best to minimize any changes you have to make.
 
+## React Native (Beta) Installation
```
Comment (NachoSoto) [inline docs_source/🛠 Tools/paywalls.md:93]: <3

**Review by NachoSoto — COMMENTED** [review]
:clap:

File: docs_source/🛠 Tools/paywalls.md (hunk 1 above)
Comment (joshdholtz) [reply docs_source/🛠 Tools/paywalls.md:9]: Ha! I thought I fixed all copy paste but... I didn't not haha 

Will fix!

File: docs_source/🛠 Tools/paywalls.md [hunk 3]
```diff
@@ -5,9 +5,17 @@ hidden: false
 ---
 RevenueCat's Paywalls allow you to remotely configure your entire paywall view without any code changes or app updates. Whether you’re building a new app, exploring new paywall concepts, or diving into experimentation; RevenueCat’s Paywalls make it easy to get started.
 
+## Paywalls are supported in the following SDK versions:
+| RevenueCat SDK           | Version required for Offering Metadata | 
+| :----------------------- | :--------------------------------------------- |
+| purchases-ios            | 4.26.0 and up                                  |
+| purchases-android        | 7.1.0 and up                                   |
+| react-native-purchases   | 7.4.0-beta.2                                   |
```
Comment (NachoSoto) [inline docs_source/🛠 Tools/paywalls.md:13]: Let's change this to beta 3 now 😇

File: docs_source/🛠 Tools/paywalls.md [hunk 4]
```diff
@@ -5,9 +5,17 @@ hidden: false
 ---
 RevenueCat's Paywalls allow you to remotely configure your entire paywall view without any code changes or app updates. Whether you’re building a new app, exploring new paywall concepts, or diving into experimentation; RevenueCat’s Paywalls make it easy to get started.
 
+## Paywalls are supported in the following SDK versions:
+| RevenueCat SDK           | Version required for Offering Metadata | 
+| :----------------------- | :--------------------------------------------- |
+| purchases-ios            | 4.26.0 and up                                  |
+| purchases-android        | 7.1.0 and up                                   |
+| react-native-purchases   | 7.4.0-beta.2                                   | 
+| purchases-flutter        | 6.3.0-beta.1                                   |
```
Comment (NachoSoto) [inline docs_source/🛠 Tools/paywalls.md:14]: Beta 2 now 😇

**Review by NachoSoto — APPROVED** [review]
<3

**Review by aboedo — APPROVED** [review]

**Pull request merged**